### PR TITLE
added Hello endpoint.

### PR DESCRIPTION
**Pull request by brckks**

### CODE REVIEW

File: src/main/java/com/emeq/counter/HelloController.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+@RestController
+public class HelloController {
+	
+	private static final String template = "Hello, %s!";
+
+	@GetMapping("/hello")
+	public Hello hello(@RequestParam(value = "name", defaultValue = "World") String name) {
+		return new Hello(String.format(template, name));
+	}
+//    http://localhost:4000/hello?name=emek
```
Comment (gungorkocak) [inline src/main/java/com/emeq/counter/HelloController.java]: @brckks We shouldn't keep commented out code/path here in code. If we need to keep it anyway, then we could add it to documentation.

File: src/main/java/com/emeq/counter/Hello.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package com.emeq.counter;
+
+public class Hello {
```
Comment (uralfrkn) [inline src/main/java/com/emeq/counter/Hello.java]: I think we can use lombok for Get and Set methods.

File: src/main/java/com/emeq/counter/Hello.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+	private final String content;
+
+	public Hello(String content) {
+		this.content = content;
+	}
+
+	public String getContent() {
+		return content;
+	}
+
```
Comment (gungorkocak) [inline src/main/java/com/emeq/counter/Hello.java]: No empty line please. No need to keep it.

File: src/main/java/com/emeq/counter/HelloController.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloController {
+	
+	private static final String template = "Hello, %s!";
+
+	@GetMapping("/hello")
+	public Hello hello(@RequestParam(value = "name", defaultValue = "World") String name) {
```
Comment (gungorkocak) [inline src/main/java/com/emeq/counter/HelloController.java]: Acceptance criteria doesn't mention that endpoint also should accept a `name` parameter. `World` string could be hardcoded in model/entity class.

File: src/main/java/com/emeq/counter/HelloController.java [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+		return new Hello(String.format(template, name));
```
Comment (uralfrkn) [inline src/main/java/com/emeq/counter/HelloController.java]: According to accepted criteria we wait to function return as `{hello: "world"}`.

**Pull request merged**